tests/Unit/Evolution/Systems/NewtonianEuler/Limiters/Test_KxrcfTci.cpp

```diff
@@ -136,6 +136,11 @@ void test_kxrcf_1d() noexcept {
   const auto neighbor_energies = make_dirmap_of_datavectors_from_value<1>(
       mesh.number_of_grid_points(), 0.);
 
+  // In realistic use of the TCI, the kxrcf_constant would be set to a finite
```

Comment:
[optional] "realistic uses" (add `s` to use?)
Below as well if you decide to change